x/concentrated-liquidity/types/constants.go

```diff
@@ -8,9 +8,9 @@ import (
 
 const (
 	// Precomputed values for min and max tick
+
+	// Tick corresponding to the at launch min spot price of 10^-12.
 	MinInitializedTick, MaxTick int64 = -108000000, 342000000
```

Comment:
This naming is temporary and will be changed on completion. Ref: https://github.com/osmosis-labs/osmosis/issues/6318

I like the suggestion and will track in #6318

For now, would prefer to keep it as is to avoid causing conflicts for the PR sequence